content/course-specific-material/introR-Remote-Apr20.Rmd

```diff
@@ -0,0 +1,77 @@
+
+### Tentative schedule
+
+Times are in Eastern time zone.
+
+**Day 1**
+
+* 10:00 am - 10:30 am – Instructors available for questions
+* 10:30 am - 11:30 am – [Introduction](/intro-curriculum/Introduction)
+* 11:30 am - 12:00 pm – [Example workflow](/intro-curriculum/data/data_workflow.Rmd)
+* 12:00 pm - 12:15 pm – Break
+* 12:15 pm - 01:30 pm – [Get](/intro-curriculum/Get)
+* 01:30 pm - 02:00 pm – Lunch
+* 02:00 pm - 02:30 pm – Finish Get
+* 02:30 pm - 03:30 pm – [Clean](/intro-curriculum/Clean)
+* 03:30 pm - 03:45 pm – Break
+* 03:45 pm - 05:00 pm – More Clean
+
+**Day 2**
+
+* 10:00 am - 10:30 am – Instructors available for questions
+* 10:30 am - 11:30 am – Finish Clean & do "Clean" startup exercise
+* 11:30 am - 12:00 pm – [Explore](/intro-curriculum/Explore)
+* 12:00 pm - 12:15 pm – Break
+* 12:15 pm - 01:30 pm – Finish explore
+* 01:30 pm - 02:00 pm - Lunch
+* 02:00 pm - 03:30 pm – [Analyze: Base](/intro-curriculum/Analyze)
+* 03:30 pm - 03:45 pm – Break
+* 03:45 pm - 05:00 pm – [Analyze: dataRetrieval](https://cran.r-project.org/web/packages/dataRetrieval/dataRetrieval.pdf) 
+
+**Day 3**
+
+* 10:00 am - 10:30 am – Instructors available for questions
+* 10:30 am - 12:00 pm – Visualize with ggplot2: [Visualize](/intro-curriculum/ggplot2/)
+* 12:00 pm - 12:15 pm – Break
+* 12:15 pm - 01:30 pm – [Repeat](/intro-curriculum/Reproduce/)
+* 01:30 pm - 02:00 pm - Lunch
+* 02:00 pm - 03:30 pm – Finish Repeat
+* 03:30 pm - 03:45 pm – Break
+* 03:45 pm - 04:30 pm – Project time: [USGS R packages](/intro-curriculum/USGS/), projects (group/individual), [Beyond Basic R blog topics](https://waterdata.usgs.gov/blog/tags/beyond-basic-r/), or [additional topics](/intro-curriculum/Additional/)
+
+### Data files
+
+Download data from the [Data page](/intro-curriculum/data/).
+
+### Additional resources
+
+* [USGS-R blog](https://waterdata.usgs.gov/blog/tags/r/)
+* [USGS-R twitter](https://twitter.com/USGS_R)
+* [USGS-R GitHub](https://github.com/USGS-R) (package source code + bug/feature reporting)
+* [RStudio cheatsheets](https://www.rstudio.com/resources/cheatsheets/) (visualization, shiny, markdown, RStudio, etc)
+* [RStudio data wrangling cheatsheet](https://www.rstudio.com/wp-content/uploads/2015/02/data-wrangling-cheatsheet.pdf)
+* [R Markdown](http://rmarkdown.rstudio.com/lesson-1.html)
+* [dataRetrieval package tutorial](https://owi.usgs.gov/R/dataRetrieval.html#1)
+
+### Instructors
+
+Patrick Eslick (peslick@usgs.gov)
+
+David Watkins (wwatkins@usgs.gov)
+
+### Lesson scripts
+
+[This zip file](https://doimspp.sharepoint.com/:u:/s/IIDDStaff/ES0hte9IfuVNgxS6Vm2xx_EBYRBpnsiK79sNGYHHaqnZ_A?e=MGxzqZ) contains the project folder and `.Rproj` file with the scripts that will be used to start the class. As the class goes on, the instructor will upload a new version with the content that they have been typing at the front during class. Instructions for downloading and unzipping will take place during the class.
```

Comment:
Let's edit this - I forgot to do that before, sorry.

> You will be emailed a zip file containing a project folder and `.Rproj` file with the scripts that will be used to start the class. As the class goes on, the instructor will be typing new code in these scripts. At the end of the class, the instructor will share their version with the content that they have been typing at the front during class. Instructions for downloading and unzipping will take place during the class.

Reply:
Just added this in my new commit